training_controller/training_controller.py

```diff
@@ -35,7 +35,7 @@
     "image_pull_policy": "Always",
     "labels": {"app": "nulog-train"},
     "restart_policy": "Never",
-    "requests": {"memory": "1Gi", "cpu": 1},
+    "requests": {},
```

Comment:
don't we need to request a GPU for training? why get rid of this?

Reply:
Line 39 is where we request a GPU.

Reply:
misread it. thanks for pointing out